documentation/ChapterArticle.json

```diff
@@ -70,20 +70,15 @@
     "reference" : {
       "type" : "Reference",
       "publicationContext" : {
-        "type" : "Journal",
-        "title" : "Tim's lovely publishing house",
-        "printIssn" : "2222-2227",
-        "onlineIssn" : "1111-1119"
+        "type" : "Chapter",
+        "partOf" : "https://123123/213123.com"
```

Comment:
Quick question: Do we want Chapters to show up when we are searching for a Book by title?